src/core/components.py

```diff
@@ -1046,13 +1046,23 @@ class Table(Component):
         table_container_width : Integer:
             Number of pixels the table wrapper will be, if specified and table is wider than the container
             Horizontal scrolling will be automatically shown on front end
+        table_filter_path: Str:
```

Comment:
Bit of a nit, but should we be describing the filtering behaviour here? Is it possible that implementations could make various uses of this input, or at least apply different matching techniques?

Reply:
I'll remove

Reply:
e1ef4bb297c77b6ebf1143a765c56ef3ac270fbc